src/Sulu/Bundle/AdminBundle/Admin/View/ListItemAction.php

```diff
@@ -23,7 +23,7 @@ class ListItemAction
     private $type;
 
     /**
-     * @var array
+     * @var array<string, string>
```

Comment:
I'm not sure if this can be even mixed as the JS components decide the options and it could there be also nested things.

```suggestion
     * @var array<string, mixed>
```